mithril.d.ts

```diff
@@ -171,10 +167,10 @@ declare namespace Mithril {
 		* @see m.mount
 		* @see m
 		*/
-		component<T extends Controller>(
-			component: Component<T>,
+		component(
```

Comment:
If controller defined as function than will error, as class all fine.
Text of error:
```
... (41,36): Error TS2322: Type 'Component<Controller>' is not assignable to type 'Component<IChartComponentController>'.
  Type 'Controller' is not assignable to type 'IChartComponentController'.
    Property 'chartName' is missing in type 'Controller'.
```
In new code i use only class controller, but this is old code. For me is not principal, but for others maybe yes.